conda_smithy/templates/azure-pipelines-osx.yml.tmpl

```diff
@@ -0,0 +1,83 @@
+jobs:
+- job: osx_64
+  pool:
+    vmImage: xcode9-macos10.13
+  strategy:
+    maxParallel: 8
+    matrix:
+    {%- for config, platform in configs | sort %}
+    {%- if platform.startswith('osx') %}
+      {{ config }}:
+        CONFIG: {{ config}}
+    {%- endif %}
+    {%- endfor %}
+
+  steps:
+  # TODO: Fast finish on azure pipelines?
+  - script: |
+      echo "Fast Finish"
+      {{ fast_finish }}
+
+  - script: |
+      echo "Removing homebrew from Travis CI to avoid conflicts."
+      curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall > ~/uninstall_homebrew
+      chmod +x ~/uninstall_homebrew
+      ~/uninstall_homebrew -fq
+      rm ~/uninstall_homebrew
+    displayName: Remove homebrew
+
+  - script: |
+      echo "Installing Miniconda"
+      set -x -e
+      curl -o $(Build.StagingDirectory)/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+      chmod +x $(Build.StagingDirectory)/miniconda.sh
+      $(Build.StagingDirectory)/miniconda.sh -b -p $(Build.StagingDirectory)/miniconda
+      export PATH=$(Build.StagingDirectory)/miniconda/bin:$PATH
+      echo "Setting up Conda environment"
+    displayName: 'Install miniconda'
```

Comment:
Is installing Miniconda necessary still necessary if version 4.5.x is [already installed](https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/macos-Readme.md#package-management) on the macOS image?

Reply:
If miniconda is already available, this could be changed to a `conda update --all`. We just need the latest version

Reply:
Think there may have been some concerns with the location of the install. Can't remember if it was specific to the the macOS image or a different image. Hopefully @mariusvniekerk remembers better than I.

Reply:
When we did this last time Miniconda was owned by something else and we couldn't alter the base environment correctly